test/providers.test.js

```diff
@@ -21,6 +21,17 @@ test('json rpc fetch block info', withProvider(async (provider) => {
     expect(sameBlock.header.height).toEqual(height);
 }));
 
+test('json rpc fetch gas price', withProvider(async (provider) => {
+    let stat = await provider.status();
+    let height = stat.sync_info.latest_block_height - 1;
+    let block = await provider.block(height);
+    let hash = block.header.hash;
+    let heightPrice, hashPrice, latestPrice = await Promise.all([provider.gasPrice(height), provider.gasPrice(hash), provider.gasPrice(null)]);
+    expect(heightPrice.gas_price).toBeGreaterThan(0);
+    expect(hashPrice.gas_price).toBeGreaterThan(0);
+    expect(latestPrice.gas_price).toBeGreaterThan(0);
```

Comment:
```js
const status = await provider.status();
for (gasPricePresponse in await Promise.all([
    provider.gasPrice(stat.sync_info.latest_block_height),
    provider.gasPrice(stat.sync_info.latest_block_hash),
    provider.gasPrice(null)
])) {
    expect(gasPriceResponse.gas_price).toBeGreaterThan(0);
}
```

Reply:
not working
<img width="448" alt="image" src="https://user-images.githubusercontent.com/33007476/75704026-73ab0580-5c6d-11ea-93d0-648161a5e6ea.png">